### ISSUE TITLE
Image segmentation in city landscape

### ISSUE THREAD

**Issue by r-luo** [r-luo/GATech-CS7643-Project-Group#4]
# Project Title

Image segmentation in city landscape

# Project summary (4-5+ sentences). 

- Fill in your problem and background/motivation (why do you want to solve it? Why is it interesting?). This should provide some detail (don't just say "I'll be working on object detection")

For self-driving car, it is necessary to separate different objects. Also, the shapes and silhouette can helps improve object tracking, resulting in a more accurate input for both steering and acceleration.

# What you will do (Approach, 4-5+ sentences)

- Be specific about what you will implement and what existing code you will use. Describe what you actually plan to implement or the experiments you might try, etc. Again, provide sufficient information describing exactly what you'll do. One of the key things to note is that just downloading code and running it on a dataset is not sufficient for a description or a project! Some thorough implementation, analysis, theory, etc. has to be done for the project.

We will be using the dataset from the link and transfer learn deep learning models. Implement different image segmentation methods, such as Unet, DeepLab. We’ll also be studying what is the performance of different models and We’ll be evaluating the performance based on Intersection of Union, Dice Coefficient. We’ll also be studying how different loss function, such as conventional cross entropy loss, focal loss, dice loss, can impact the results.

# Resources / Related Work & Papers (4-5+ sentences). 
- What is the state of art for this problem? Note that it is perfectly fine for this project to implement approaches that already exist. This part should show you've done some research about what approaches exist.

Deep Lean based segmentation models are used for image segmentation for self-driving scenario.  Such as U net, FU net, Deep Lab, etc. 

## Papers:

https://ieeexplore.ieee.org/abstract/document/9356353?casa_token=ubIvqwRFp0sAAAAA:qT3J0N-7lpxAED0WFAdf8KkRnW8tZ-u-Zei0ZEq2aT-8-qkm6KN0S2SXSSThVHyKMIWncuvt
https://link.springer.com/chapter/10.1007/978-981-15-8391-9_1

# Datasets (Provide a Link to the dataset). 
- This is crucial! Deep learning is data-driven, so what datasets you use is crucial. One of the key things is to make sure you don't try to create and especially annotate your own data! Otherwise the project will be taken over by this.

https://www.kaggle.com/c/cvpr-2018-autonomous-driving

List your Group members.
Are you looking for more members?
